data/data/azure/main.tf

```diff
@@ -45,9 +45,8 @@ module "bootstrap" {
   private                = module.vnet.private
   outbound_udr           = var.azure_outbound_user_defined_routing
 
-  use_ipv4                  = var.use_ipv4 || var.azure_emulate_single_stack_ipv6
+  use_ipv4                  = var.use_ipv4
```

Comment:
The extra whitespace before the `=` needs to be remove to appease the format checkers.
You can update the formatting with the following command.
```
podman run --rm \
 --env IS_CONTAINER=TRUE \
 --volume "${PWD}:${PWD}:z" \
 --workdir "${PWD}" \
 quay.io/coreos/terraform-alpine:v0.12.0-rc1 \
 terraform fmt -write=true -recursive data/data
```

Reply:
fixed